getting_started.rst

```diff
@@ -235,48 +220,31 @@ You can generate a graph of your dependencies, in dot or html formats:
     :width: 200 px
     :align: center
 
+
 Building with other configurations
 ----------------------------------
+As an exercise, try building your timer project with a different configuration.
+For example, you could try building the 32 bits version.
 
-In this example we have built our project using the default configuration detected by conan, this
-configuration is known as the :ref:`default profile<default_profile>`.
-
-The first time you run the **conan** command, your settings are detected (compiler,
-architecture...) automatically and stored as default in a profile. You can change your those
-settings by editing ``~/.conan/profiles/default`` or create new profiles with the desired
-configuration.
-
-For example, if we have a profile with a gcc configutarion for 32 bits in a profile called
```

Comment:
You are removing some valid contents that were added recently, in order to solve the conflicts.
Don't worry about this one, I can add back those contents later.